editor/Dockerfile

```diff
@@ -119,22 +119,39 @@ RUN [ `echo $version-$module | grep -v '^\(2018.3\|2018.4\).*-android'` ] && exi
     echo ''; \
     echo 'xvfb-run -ae /dev/stdout "$UNITY_PATH/Editor/Unity" -batchmode "$@"'; \
   } > /usr/bin/unity-editor \
-  && chmod +x /usr/bin/unity-editor
+  && chmod +x /usr/bin/unity-editor \
+  && { \
+    echo "export ANDROID_SDK_ROOT=${ANDROID_SDK_ROOT}"; \
+    echo "export ANDROID_HOME=${ANDROID_HOME}"; \
+    echo "export ANDROID_NDK_HOME=${ANDROID_NDK_HOME}"; \
+    echo "export JAVA_HOME=${JAVA_HOME}"; \
+    echo "export PATH=${PATH}"; \
+  } > ~/.bashrc
 
 #=======================================================================================
 # [2019.x/2020.x/2021.x-android] Support usage of installed Android SDKs outside of editor
 #=======================================================================================
-RUN [ `echo $version-$module | grep -v '^20(?!18).*-android'` ] && exit 0 || : \
+RUN [ `echo $version-$module | grep -vP '^20(?!18).*-android'` ] && exit 0 || : \
+  \
+  # Install jq
+  && apt-get update -qq \
+  && apt-get install -qq -y --no-install-recommends jq \
   \
   # Environment Variables
   && export RAW_ANDROID_SDK_ROOT=$(jq -cr '(.[] | select(.id == "android-sdk-platform-tools")).destination' $UNITY_PATH/modules.json) \
-  && export ANDROID_SDK_ROOT=${RAW_ANDROID_SDK_ROOT/'{UNITY_PATH}'/$UNITY_PATH} \
+  && export ESCAPED_UNITY_PATH=$(printf '%s' "$UNITY_PATH" | sed 's/[#\/]/\\\0/g') \
+  && export ANDROID_SDK_ROOT=$(echo $RAW_ANDROID_SDK_ROOT | sed -e "s/{UNITY_PATH}/$ESCAPED_UNITY_PATH/g") \
```

Comment:
Since we aren't in a full bash environment, jump through some sed hoops to do the replacement.

Reply:
You seem to know a lot about bash scripting. 🚀

Do you have recommended reading for me to learn more about what it means to have a "full bash environment" and how to find out? 

I think I'm having a knowledge gap there.

Reply:
I'm no expert. I just know enough to let Google fill in the blanks.

Docker run uses a login less shell which defaults to `/bin/sh` which actually symlinks to `dash`. Logging in normally will default to `bash`.
Here is a simple breakdown of the [2 shells](https://blog.cloudware.bg/en/dash-vs-bash-shell/).